src/diffusers/models/unet_2d_condition.py

```diff
@@ -52,6 +52,54 @@
 
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
+@dataclass
+class UNetFreeUConfig:
+    enabled: bool = False
+    s1: float = 1.0
+    s2: float = 1.0
+    b1: float = 1.0
+    b2: float = 1.0
+
+    def sd21(self):
+        self.s1 = 0.9
+        self.s2 = 0.2
+        self.b1 = 1.1
+        self.b2 = 1.2
+
+    def ones(self):
+        self.s1 = 1.0
+        self.s2 = 1.0
+        self.b1 = 1.0
+        self.b2 = 1.0
+        
+def fourier_filter(x_in, threshold, scale):
```

Comment:
Could we put this into the PyTorch utils file here: https://github.com/huggingface/diffusers/blob/main/src/diffusers/utils/torch_utils.py